### PR TITLE
Add translations and sync the locales

### PR DESCRIPTION
**Pull request by 1pretz1**
Imports translations from the supplier (ASP). As part of this work, the translations have also been audited and synced.

More info in the commits ->

Trello:
https://trello.com/c/Xz78vNR4/2757-import-newly-sourced-translations-into-locale-files-5

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -247,7 +247,7 @@ fr:
       delivery_title:
         minister: Ministre
         speaker: Orateur
-      written_on: 'Rédiger le: '
+      written_on: 'Rédiger le :'
```
Comment (MuriloDalRi) [inline config/locales/fr.yml]: This seems weird, is it a special case?

**Review by MuriloDalRi — COMMENTED** [review]
This looks good, I'm just not sure why we need to remove unused keys, is that step necessary? Don't we want to keep those translations?

**Comment by 1pretz1** [comment]
Thanks for the review @MuriloDalRi! 

> I'm just not sure why we need to remove unused keys, is that step necessary? Don't we want to keep those translations?

It's not strictly necessary to remove unused keys as part of the import, but I thought it would be useful to remove them, rather than add keys that are not actually used in the application. If we were to add them then this would create more work for us when we come to validate the locales. It is sad that the locale files have changed since we sent the files off and quite a few of them are no longer needed.

File: config/locales/fr.yml (hunk 1 above)
Comment (MuriloDalRi) [reply config/locales/fr.yml]: Oh wow, I've learned something new today! :D

**Review by MuriloDalRi — APPROVED** [review]

File: config/locales/fr.yml (hunk 1 above)
Comment (ChrisBAshton) [reply config/locales/fr.yml]: +1!

**Pull request merged**